<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficViolationRecord</class>
 <widget class="QWidget" name="TrafficViolationRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>490</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查询车牌号：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditCar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuery">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
    <property name="icon">
     <iconset resource="images/myimages.qrc">
      <normaloff>:/new/imgsrc/Look.png</normaloff>:/new/imgsrc/Look.png</iconset>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditStart">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2013</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2013</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditStop">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2017</year>
      <month>10</month>
      <day>10</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2013</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>结束时间：</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>刷新内容</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/myimages.qrc">
      <normaloff>:/new/imgsrc/re.png</normaloff>:/new/imgsrc/re.png</iconset>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>731</width>
     <height>431</height>
    </rect>
   </property>
   <widget class="QTableView" name="tableView_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>431</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>361</width>
      <height>431</height>
     </size>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>361</width>
      <height>431</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>361</width>
      <height>431</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images/myimages.qrc"/>
 </resources>
 <connections/>
</ui>
